debezium-connector-sqlserver/src/test/java/io/debezium/connector/sqlserver/SqlServerConnectorTest.java

```diff
@@ -43,6 +45,78 @@ public void testValidateUnableToConnectNoThrow() {
         }
     }
 
+    @Test
+    public void shouldReturnSingleTaskConfig() {
+        Map<String, String> config = new HashMap<>();
+        config.put(SqlServerConnectorConfig.HOSTNAME.name(), "narnia");
+        config.put(SqlServerConnectorConfig.PORT.name(), "4321");
+        config.put(SqlServerConnectorConfig.DATABASE_NAME.name(), "sqlserver1");
+        config.put(SqlServerConnectorConfig.DATABASE_NAMES.name(), "sqlserver1,sqlserver2");
```

Comment:
Should it be `database1,database2` instead? Otherwise, may be confusing.

Reply:
I can update. I was just following the convention of the other unit tests already present.